src/textual/screen.py

```diff
@@ -18,8 +18,8 @@
 from .widget import Widget
 
 
-# Screen updates will be batched so that they don't happen more often than 60 times per second:
-UPDATE_PERIOD: Final = 1 / 60
+# Screen updates will be batched so that they don't happen more often than 120 times per second:
```

Comment:
Worth turning the comment into a docstring?

Reply:
Don't think so. No reason for this to be in docs.

Reply:
That makes sense, although don't gorget that docstrings serve anyone working on the code as any good editor setup will show the documentation when focus is around that item (personally I try and write docstrings with that purpose in mind -- which is presumably why we also do them for `_private` methods and the like too?).